content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -127,7 +127,7 @@ In release 1.8, quota support for local ephemeral storage is added as an alpha f
 ## Object Count Quota
 
 You can set quota for the total number of certain resources of all standard,
-namespaced resource types using the following syntax:
```

Comment:
I think this change might not be required. Can you specifically add a link for the document you're referring to?

Reply:
line 535 in this document

Reply:
I don't think we need this change either.

Reply:
I don't believe this change is needed either